### PR TITLE
Remove QGEP and fix teksi wastewater urls

### PR DESCRIPTION
**Pull request by ponceta**
- [x] Fix https://github.com/teksi/wastewater/issues/262

### CODE REVIEW

File: plugin/teksi_wastewater/metadata.txt [hunk 1]
```diff
@@ -1,19 +1,19 @@
 
 [general]
 name=TEKSI Wastewater
-description=Plugin to manage wastewater networks (formerly: QGEP) # // skip-keyword-check
-qgisMinimumVersion=3.2
+description=TEKSI Wastewater plugin to manage wastewater networks. Provide selection, connection and editing tools.
+qgisMinimumVersion=3.34
 category=Vector
 version=0.0.0
```
Comment (sjib) [inline plugin/teksi_wastewater/metadata.txt:7]: When do we add the version 2024.0?

File: plugin/teksi_wastewater/metadata.txt [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
 changelog=See repository for full changelog
-tags=teksi, gep, abwasser, wastewater, qgep, assainissement # // skip-keyword-check
+tags=teksi, gep, abwasser, wastewater, assainissement
```
Comment (sjib) [inline plugin/teksi_wastewater/metadata.txt:10]: add also these tags:
network, pgee, sewer, infrastructure, management 
as tags?

File: plugin/teksi_wastewater/metadata.txt [hunk 3]
```diff
@@ -1,19 +1,19 @@
 
 [general]
 name=TEKSI Wastewater
-description=Plugin to manage wastewater networks (formerly: QGEP) # // skip-keyword-check
-qgisMinimumVersion=3.2
+description=TEKSI Wastewater plugin to manage wastewater networks. Provide selection, connection and editing tools.
```
Comment (sjib) [inline plugin/teksi_wastewater/metadata.txt:4]: In qgep we had the following text: The QGEP plugin adds functionality to manage wastewater networks in QGIS. It requires a QGEP compatible database and project file.

Do we want to add the second part again?
Do we want to add something as - It supports Swiss standards SIA405 wastewater, VSA-DSS and VSA-KEK  (2020.x) for INTERLIS import / export. Or would this something to be added on the homepage?

File: plugin/teksi_wastewater/metadata.txt [hunk 4]
```diff
@@ -1,13 +1,14 @@
 
 [general]
 name=TEKSI Wastewater
-description=TEKSI Wastewater plugin to manage wastewater networks. Provide selection, connection and editing tools.
+description=TEKSI Wastewater plugin to manage wastewater networks. Provide selection, connection and editing tools. Require a compatible TEKSI wastewater datamodel and project file. Support Swiss standards SIA405 wastewater, VSA-DSS and VSA-KEK (2020.x) for INTERLIS import / export.
```
Comment (sjib) [inline plugin/teksi_wastewater/metadata.txt:4]: description=TEKSI Wastewater plugin to manage wastewater networks. Provides selection, connection and editing tools. Requires a compatible TEKSI wastewater datamodel and project file. Supports Swiss standards SIA405 wastewater, VSA-DSS and VSA-KEK (2020.x) for INTERLIS import / export.

Comment (sjib) [reply plugin/teksi_wastewater/metadata.txt:4]: Thanks @ponceta

**Review by cymed — APPROVED** [review]

**Pull request merged**